### PR TITLE
fix(stryker-cli.ts): Execute stryker init after install

### LINKED ISSUE #20: Command is not passed to stryker if it has to be installed
Repro steps:
* Create a new directory
* Call `npm init` and skip through it
* Call `stryker init` and choose to install stryker

Expected behavior:
Stryker is installed locally and the `init` command is passed to it, starting the initializer

Current behavior:
Stryker is installed locally and you are suggested to run `stryker --help`:
```
Stryker installation done.
Get started by using `stryker --help`.
```

Fix:
Call `run()` after stryker has been installed. Merge #18 first.

### PR DESCRIPTION
**Pull request by korthout**
Fix #20 by executing `stryker init` after Stryker was installed locally.

### CODE REVIEW

**Comment by korthout** [comment]
#20 proposed to call `run()` after the installation, however executing stryker init asynchronously results in a nicer console output.

**Comment by korthout** [comment]
I've removed the console output from the installation part, which wasn't necessary in the first place and allows for the cleaner use of `run()` instead of `execSync('stryker init')`.

**Comment by korthout** [comment]
@nicojs @avassem85 I think this PR is ready for review. If you have some time available, please take a look at it. :)

File: src/stryker-cli.ts [hunk 1]
```diff
@@ -43,16 +51,15 @@ function promptInstallStryker() {
...
     }
   });
 }
 
 function installStryker() {
   printStrykerASCII();
-  child.execSync('npm i --save-dev stryker stryker-api', { stdio: [0, 1, 2] });
-  console.log(chalk.green('Stryker installation done.'));
-  console.log(`Get started by using ${chalk.blue('`stryker --help`')}.`);
+  executeInstallStrykerProcess();
```
Comment (nicojs) [inline src/stryker-cli.ts]: Code complete. I can see that 👍

**Pull request merged**